### PR TITLE
README.md: fix the badge

### LINKED ISSUE #53: Fix CI banner in README.md
Seems like the tP tracker pointed out that the CI banner in the `README.md` isn't pointing to the correct repo.

See below for details:
image

### PR DESCRIPTION
**Pull request by jieffyy**
Closes #53.

Update the badge link based on the tP tracker's feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2021S1-CS2103T-T11-2/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (yanchenglee98) [inline README.md]: need to change https://github.com/se-edu/addressbook-level3/actions to this https://github.com/AY2021S1-CS2103T-T11-2/tp/actions

**Review by yanchenglee98 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jieffyy) [reply README.md]: Hmm the old link generates a badge though. Using `/actions/` doesn't generate a badge, but a link instead...

Comment (jieffyy) [reply README.md]: Ohh I see. Sorry I misunderstood that.

**Review by yanchenglee98 — APPROVED** [review]
LGTM

**Pull request merged**